Lib/distutils/tests/test_dist.py

```diff
@@ -490,6 +490,14 @@ def test_show_help(self):
                   if line.strip() != '']
         self.assertTrue(output)
 
+    def test_parse_command_opts(self):
+        dist = Distribution()
+        sys.argv = ["setup.py", "build", "--quiet"]
+        dist.parse_command_line()
+        # Note: the `quiet` option is an alias that negatively affects the
+        # `verbose` option value, that is `True` by default
+        self.assertFalse(dist.verbose)
```

Comment:
Can you also assert that `dist.quiet` is `True`?

Reply:
`dist.quiet` doesn't exist. As I explained in the comment `quiet` is just an option that negatively affects the `verbose` option.